internal/domain/power-status.go

```diff
@@ -358,6 +358,29 @@ func updateComponentMap() error {
 					newComp.HSMData.PowerCapURI = v.PowerCapURI
 					newComp.PSComp.LastUpdated = time.Now().Format(time.RFC3339)
 					hwStateMap[v.BaseData.ID] = &newComp
+				} else {
+					// This is an existing component, update if necessary
+					if xnametypes.HMSType(v.BaseData.Type) == xnametypes.NodeBMC &&
+					   len(hwStateMap[v.BaseData.ID].PSComp.SupportedPowerTransitions) == 0 &&
+					   len(v.AllowableActions) > 0 {
+
+						// Some node BMCs require updating their supported
+						// power transitions.  Node BMCs never change power
```

Comment:
"BMCs require updating their supported power transitions"
Does this only happen after a firmware update?